app/views/speech/show.html.erb

```diff
@@ -0,0 +1,58 @@
+
+<% content_for :extra_headers do %>
+   <%= render "govuk_publishing_components/components/machine_readable_metadata", { content_item: @content_item.to_h, schema: :article } %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/title', {
+        title: content_item.title,
+        context: I18n.t("csv_preview.document_type.#{content_item.document_type}", count: 1),
+        context_locale: t_locale_fallback("csv_preview.document_type.#{content_item.document_type}", count: 1),
+        average_title_length: "long",
+      } %>
+  </div>
+  <%= render 'shared/translations' %>
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/lead_paragraph', text: @content_item.description %>
+  </div>
+</div>
+
+<%= render 'shared/publisher_metadata_with_logo' %>
+<%= render 'shared/history_notice', content_item: @content_item %>
+<% if @content_item.withdrawn? %>
+  <%= render 'govuk_publishing_components/components/notice', @content_item.withdrawal_notice_component  %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="content-bottom-margin">
+      <div class="responsive-bottom-margin">
+        <%= render 'components/important_metadata',
+        items: @content_item.important_metadata %>
+
+        <%= render 'components/figure',
+          src: @content_item.image["url"],
+          alt: @content_item.image["alt_text"],
+          credit: @content_item.image["credit"],
+          caption: @content_item.image["caption"] if @content_item.image %>
+
+        <%= render "govuk_publishing_components/components/govspeak", {
+          direction: page_text_direction,
+        } do %>
+          <%= raw(@content_item.body) %>
+        <% end %>
+      </div>
+
+      <%= render 'components/published_dates', {
+          published: display_date(@content_item.first_published_at),
```

Comment:
The variable used here is now `first_public_at`, similar to https://github.com/alphagov/frontend/pull/4372/commits/6c5ed23d73e9103cb723e51d7e08bdfd29b64089#diff-de0d946d1ce699f8a4070b4d6a67cc44726747222c15044f2bc7e00e3a314743R60